docs/operate-and-deploy/installation/upgrading.md

```diff
@@ -615,4 +615,26 @@ SELECT * FROM input PARTITION BY col0 EMIT CHANGES;
 ### `ALL`, `WINDOWSTART` and `WINDOWEND` are now reserved identifiers
 
 Any query using these identifiers will need to be changed to either use some other identifier, or
-to quote them.
\ No newline at end of file
+to quote them.
+
+
+
+### Upgrading from confluentksqlDB 5.5 to confluentksqlDB 6
+
+In-place upgrades are supported from confluentksqlDB 5.5 to confluentksqlDB 6.
+
+#### Steps of "In-Place" upgrade :-
+
+1. Download the latest confluent version.
```

Comment:
```suggestion
1. Download the new ksqlDB version to upgrade to.
```
The new version need not be a Confluent ksqlDB version, the community versions work as well with the same instructions.

Reply:
Might be helpful to provide a link.

Reply:
@JimGalasyn I have added a download link. Can you check if that makes sense ?